src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Windows.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibGpiodDriver : UnixDriver
```

Comment:
is UnixDriver much different from IGpioDriver? I'm wondering if we should instead have something like NotImplementedDriver and reuse it in places like this to reduce lines of code

Reply:
Seems we only have IGpioController but no interface for driver. Same question but GpioDriver

Reply:
> I'm wondering if we should instead have something like NotImplementedDriver and reuse it in places like this to reduce lines of code

That could work, we can address that in a different PR.
About UnixDriver vs GpioDriver, they are pretty much the same, except that in Linux UnixDriver class has some implementation and the reference assembly has to be produced out of that, which means that in here we can't derive from GpioDriver, and have to derive from UnixDriver instead in order to be compatible.